Makefile

```diff
@@ -68,6 +73,10 @@ gofail-disable: install-gofail
 install-gofail:
 	go install go.etcd.io/gofail
 
+.PHONY: install-goimports
+install-goimports:
+	go install golang.org/x/tools/cmd/goimports@latest
```

Comment:
If we called directly: `go run golang.org/x/tools/cmd/goimports@latest` in `fmt`
there would be no need to install. Golang would cache the newest version.

Reply:
Good point, much simpler, updated.